AzureRmStorageTableCoreHelper.psm1

```diff
@@ -480,6 +480,8 @@ function Get-AzTableRow
 		Identifies the table partition (byPartitionKey and byPartRowKeys parameter sets)
 	.PARAMETER RowKey
 		Identifies the row key in the partition (byPartRowKeys parameter set)
+	.PARAMETER SelectColumns

```

Comment:
Lets rename this to SelectColumn (singular), from Powershell perspective we don't use plurals as parameter names and cmdlet names. Please apply to all ocurrences of this.

Reply:
Sounds good, I adjusted this across the board.